menshen/server.js

```diff
@@ -56,23 +62,34 @@ async function main() {
 			})
 			.on("error", function (err) {
 				socket.emit("data", `\r\n=== [SERVER] ERROR: ${err.message} ===\r\n`);
-			})
-			if (!config._query["sshPrivateKey"]) {
-				conn.connect({
-					host: config._query["sshHost"],
-					port: config._query["sshPort"],
-					username: config._query["sshUsername"],
-					password: config._query["sshPassword"],
-				});	
-			} else {
-				conn.connect({
-					host: config._query["sshHost"],
-					port: config._query["sshPort"],
-					username: config._query["sshUsername"],
-					password: config._query["sshPassword"],
-					privateKey: config._query["sshPrivateKey"],	
-				});
-			}
+			});
+
+		socket.on("disconnect", function () {
+			console.log(`\r\n=== [SERVER] Connection to Client Terminated ===\r\n`);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/KingWaffleIII/kommander/pullRequest?prid=9134818)
```suggestion
			console.log("\r\n=== [SERVER] Connection to Client Terminated ===\r\n");
```